src/widgets/JSONForm/templates/Field.js

```diff
@@ -60,28 +60,33 @@ export const Field = ({
 
   if (schemaIsRoot(id)) {
     return (
-      <ScrollView keyboardDismissMode="none" keyboardShouldPersistTaps="always">
-        {Content}
-      </ScrollView>
+      <Paper
```

Comment:
Changes here are to make the form a single paper, rather than a paper per section..? I liked the more explicit section differences but then I didn't know where to put the 'main' header 🤔 

Maybe it would be better to just have explicit paper per section and not have a "Extra info" header?

Reply:
I liked the paper-per-section, but agree that the main header looks good like this. Really ties the two forms together better.
Tried background and border on the heading, but couldn't make it look better than it does now.